### PR TITLE
TF precommit on CPU

### PR DESCRIPTION
**Pull request by negvet**
### Changes

Graph test updated to support CPU device.

### Reason for changes

Support pre-commit on CPU

### Related tickets

76181

### CODE REVIEW

**Review by daniil-lyakhov — APPROVED** [review]

**Comment by daniil-lyakhov** [comment]
As an alternative we could force our graph tests to use only CPU version of graphs

**Comment by negvet** [comment]
> As an alternative we could force our graph tests to use only CPU on run

AFAIK we have a separate pre-commit that run only on CPU

File: tests/tensorflow/test_compressed_graph.py [hunk 1]
```diff
@@ -232,6 +232,19 @@ def __init__(self, ref_graph_filename: str, model_builder,
 }
 
 
+gpu_available = len(tf.config.list_physical_devices('GPU')) > 0
+if gpu_available:
+    SKIP_MAP['quantization'].update({'inception_v3_cpu': pytest.mark.skip(reason='cpu version of the graph')})
```
[the remark this answers is not in the document]
Comment (negvet) [reply tests/tensorflow/test_compressed_graph.py]: Indeed we can. Thanks for the comment. Updated.

**Comment by negvet** [comment]
Jenkins please retry a build

**Comment by daniil-lyakhov** [comment]
Please rebase on new inception_v3 graphs

**Review by alexsu52 — COMMENTED** [review]
Please, provide test results for CPU pre-commit.

**Comment by negvet** [comment]
Jenkins please retry a build

**Pull request merged**